### PR TITLE
book: Talk about ignore attribute in testing guide

### PR DESCRIPTION
**Pull request by adamcrume**

### CODE REVIEW

File: src/doc/trpl/testing.md [hunk 1]
```diff
@@ -219,6 +219,63 @@ fn it_works() {
 This is a very common use of `assert_eq!`: call some function with
 some known arguments and compare it to the expected output.
 
+# The `ignore` attribute
+
+Sometimes a few specific tests can be very time-consuming to execute.  These
+can be disabled by default by using the `ignore` attribute:
```
Comment (killercup) [inline src/doc/trpl/testing.md]: > execute.  These

Is there a precedence in the book for double spaces after sentence ends?

**Comment by killercup** [comment]
Good addition. I'd probably try to explain that `--ignored` is a flag on the test binary (not `cargo`) to avoid confusion over why there is an additional `--` between `test` and the argument.

File: src/doc/trpl/testing.md (hunk 1 above)
Comment (steveklabnik) [inline src/doc/trpl/testing.md]: There should be only one space.

**Comment by steveklabnik** [comment]
@adamcrume Both of @killercup 's points are valid, but I think they're small enough that if you weren't willing to make them, we can take this as-is anyway. Let me know what you think either way :)

**Comment by adamcrume** [comment]
@steveklabnik I'm happy to make the changes.  I'll have free time to work on it this evening.

**Comment by killercup** [comment]
LGTM, @adamcrume.

**Comment by steveklabnik** [comment]
Thanks so much!

**Pull request merged**